### ISSUE TITLE
[bug] StructureFromMotion crashes @ "Compute scene structure color" phase 

### ISSUE THREAD

**Issue by kmlbgn** [alicevision/Meshroom#1979]
Meshroom crashes at the very end of the StructureFromMotion node, when `Compute scene structure color` appears. 

This seem related to #183 but is actually not because the example shown in this issue actually have the "sfm.abc" file being exported. 
In my case, it just crashes there, and I'm left with the intermediate steps .abc files. 
```
(...) 

Compute scene structure color
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|

( crashes )
``` 

![image](https://user-images.githubusercontent.com/59659182/231830220-d9bff003-1959-4158-972b-c3b3dc38f3b3.png)

**Comment by kmlbgn** [comment]
I run the computation with `trace` verbose level
Seems like some error when reading one of the tif. 

```
Compute scene structure color
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-125.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-025.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-231.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-061.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-050.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-134.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-109.tif
[22:19:26.390662][debug] [IO] Read Image: C:/Users/Kamil/Desktop/Sumbawa Rhino 3D/FFMPEG/DJI_0859/image-047.tif
```

**Comment by kmlbgn** [comment]
Yes, there is no way to delete the first node `CameraInt` and replace it by new one nor replacing the pictures and renaming the sfmdata folder to the original (already computed one). Which means i have to recompute everything unless I missed something. 

If this is the case, it's pretty disappointing guys. The pipeline workflow is a very good idea and it should definitely allow to leverage its modularity by allowing manual setup of SFMdata and feature folders in case something goes wrong and e want to reuse the already computed data.

**Comment by mugulmd** [comment]
Hi @kmlbgn !

I don't think your problem is related to spaces, we've already had that bug but we haven't found the time to solve it yet.
However you can always disable the computation of the scene structure color, if you click on the `StructureFromMotion` node you have an option for that in the attribute editor 😄 

As for removing links between nodes, you have a "Remove" option if you right-click on the link (but I'm definitely taking your remark into account, we should add some more intuitive way to delete the links).

**Comment by kmlbgn** [comment]
The problem was that I had to move my folder to a external SDD because of #1969 , and so the structure color where looking for the original file path which, I suppose, are contained in the first node that I tried to change. Thanks for the info though i'll look more into the parameters next time. 

Basically I just wanted to delete all images and replace with new ones. But the problem i'm pointing in my previous message is that your interface is rather rigid and don't allow this kind of manipulation. In my case, the image are the same so no recomputation needed. So I would recompute node 1 and just tell the next node to pick data from previous feature folder. 

Also guys, unless i missed something again, it is not possible to bulk delete images !! We cannot select bunch of them, nor use SHIFT or anything. I had to delete my clicking the delete button for 5 min (256 pic)

**Comment by cbentejac** [comment]
Hi @kmlbgn,

Jumping in on that comment:
> Also guys, unless i missed something again, it is not possible to bulk delete images !! We cannot select bunch of them, nor use SHIFT or anything. I had to delete by clicking the delete button for 5 min (256 pic)

There actually is an option to clear the Image Gallery easily. In the "File" menu, the  "Clear Images" option will remove all the images in just one click.

Hope that helps!

**Comment by kmlbgn** [comment]
Can you cherry pick ? Or its all or nothing

**Comment by cbentejac** [comment]
You can't cherry-pick: it deletes all the images. 

There currently isn't any option that allows you to select a few images and remove them all at once, but you can navigate the gallery using the keyboard's arrows and remove the selected image with the "Delete" key, which is more convenient (and faster) than right-clicking and selecting "Remove" every time.

**Comment by fabiencastan** [comment]
In the meantime, you can disable the color estimation as it is here only for visualization and it is useless in the pipeline.
